gp-populate-anything/gppa-force-dyn-pop-on-gv-edit.php

```diff
@@ -7,10 +7,19 @@
  * https://github.com/gravitywiz/snippet-library/blob/master/gp-populate-anything/gppa-gpnf-force-dyn-pop-on-gv-edit.php
  */
 add_filter( 'gravityview/edit_entry/field_value', function( $field_value, $field ) {
+	$gppa_version = defined( 'GPPA_VERSION' ) ? GPPA_VERSION : 'NA';
+	// If GPPA is not active, return.
+	if ( $gppa_version === 'NA' ) {
+		return $field_value;
+	}
+
 	// Update "123" to your form ID and "4" to your field ID.
 	if ( $field->formId == 123 && $field->id == 4 ) {
 		if ( isset( $GLOBALS['gppa-field-values'] ) ) {
-			$hydrated_field = gp_populate_anything()->hydrate_field( $field, GFAPI::get_form( $field->formId ), array(), null, false );
+			// Based on GPPA version, we will need a different method call.
+			$hydrated_field = version_compare( $gppa_version, '2.0', '>=' ) ?
```

Comment:
Worth noting, we haven't typically been attempting to maintain backwards compatibility with snippets and previous versions of GPPA.

Reply:
Noted. Updated!